src/Evolution/EventsAndTriggers/EventsAndTriggers.hpp

```diff
@@ -75,14 +75,4 @@ namespace OptionTags {
 /// \cond
 struct EventsAndTriggersTagBase {};
 /// \endcond
-
```

Comment:
move the base tag too?

Reply:
This tag used by non-evolution-specific code (RunEventsAndTriggers) and we want to retrieve the E+T in the same was for all the types of problems.  We will likely want multiple EventsAndTriggers tags inheriting off this base tag, probably differing only by help string.

Reply:
okay